reps/serve_pipeline.md

```diff
@@ -0,0 +1,139 @@
+
```

Comment:
Could we add a date to this file name? Such as YYYY-MM-DD-title.md, so `2022-03-08-serve_pipeline.md`

That will ensure a sensible sort order in the github listing.